File: ExemploJSONParser/app/src/main/java/nuvemapp/com/br/exemplojsonparser/Potencia.java
package nuvemapp.com.br.exemplojsonparser;

/**
 * Created by XPredator on 29/02/2016.
 */
public class Potencia {

    private double motor;
    private int cavalos;

    public Potencia(){
        super();
    }
    public Potencia(float motor, int cavalos){
        this.motor = motor;
        this.cavalos = cavalos;
    }


    public double getMotor() {
        return motor;
    }
    public void setMotor(double motor) {
        this.motor = motor;
    }


    public int getCavalos() {
        return cavalos;
    }
    public void setCavalos(int cavalos) {
        this.cavalos = cavalos;
    }
}

File: ExemploJSONParser/PHP/Potencia.php
<?php
	class Potencia {
		private $motor;
		private $cavalos;
		
		
		public function __construct($motor=0, $cavalos=0){
			$this->motor = $motor;
			$this->cavalos = $cavalos;
		}
		
		
		public function getMotor(){
			return($this->motor);
		}
		public function setMotor($motor){
			$this->motor = $motor;
		}
		
		
		public function setCavalos($cavalos){
			$this->cavalos = $cavalos;
		}
		public function getCavalos(){
			return($this->cavalos);
		}
		
		
		public function getDataJSON(){
			$aux = array(
				'motor'=>$this->motor,
				'cavalos'=>$this->cavalos);
			
			return($aux);
		}
	}
?>